### ISSUE TITLE
cannot insert image into upgraded GOTS

### ISSUE THREAD

**Issue by obidm** [ualibraries/Guide-on-the-Side#147]
I recently copied our current GOTS installation and upgraded it as a test. While trying to create new tutorials within the upgraded GOTS, no users can insert images - there's no error coming up, it just doesn't work. Nothing happens after you select a file for upload or put in the URL of one online. Also, some users, but not all, have trouble applying text effects like bold and underlline.

Any ideas on how to diagnose this? Apache2 logs don't seem to show anything amiss when these problems are expressed, and I couldn't find any logs in the cake log directory at all.

**Comment by Blake-** [comment]
Did you check the permissions on app/webroot/uploads ?

**Comment by obidm** [comment]
Yes. It is a copy of a working gots, so nothing was changed. I manually
checked of course, it's 755 from there down, for www-data, which is Apache2.

What else should I check?

Thank you,
Doug Simmons

Did you check the permissions on app/webroot/uploads ?

—
Reply to this email directly or view it on GitHub
<https://github.com/ualibraries/Guide-on-the-Side/issues/147#issuecomment-184492778>
.

**Comment by Blake-** [comment]
darn, thought for sure that would be it. Can you switch on error display or logging for php? There's also an error log or two you can enable and have it write to ./app/tmp/logs/ if it's not already in app/Config/core.php

**Comment by obidm** [comment]
at present in core.php I have this: define('LOG_ERROR', 2);

Maybe the fact that I can't find gots logs anywhere is an indication that I
do have permissions issues somewhere.

I am not sure I want more verbose error logs from PHP, this production
server is pretty busy and there's a lot of other PHP apps running. How
likely do you think it would be that I would suddenly start getting
gigabyte logs filling up my server?

Thanks for helping me with this,
Doug Simmons

**Comment by Blake-** [comment]
eh, not likely, flip it on and then flip it back off as soon as you see something interesting.

**Comment by caosborne89** [comment]
Hello obidm,
What version of php are you using?
